modules/javafx.graphics/src/main/java/javafx/scene/input/PickResult.java

```diff
@@ -203,6 +203,7 @@ public String toString() {
         if (getIntersectedTexCoord() != null) {
                 sb.append(", texCoord = ").append(getIntersectedTexCoord());
```

Comment:
Can you fix the double indentation in the `if` bodies?

Reply:
Sure, as long as I am there, I'll do that.